project.clj

```diff
@@ -3,16 +3,17 @@
   :url "https://github.com/derwolfe/fernet-clj"
   :license {:name "MIT"
             :url "https://raw.github.com/derwolfe/fernet-clj/master/LICENSE"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 [org.bouncycastle/bcprov-jdk15on "1.49"]
-                 [commons-codec/commons-codec "1.8"]
-                 [clojurewerkz/buffy "0.3.0"]]
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.bouncycastle/bcprov-jdk15on "1.54"]
+                 [commons-codec/commons-codec "1.10"]
+                 [clojurewerkz/buffy "1.0.2"]]
```

Comment:
To be honest, I've not had to really any of the legwork involving buffy - that was entirely done by David. Overall, it has *looked* like a pleasant library with which to work.